### DISCUSSION TITLE
Discussion

### DISCUSSION THREAD

**Discussion by Prikshit7766** [RakshitKhajuria/ice-breaker#5] [Q&A] [answered]
what is this repo about , can you explain in detail  ?

**Comment by RakshitKhajuria** [comment, marked as answer]
This is a web application designed to crawl Linkedin and Twitter data about a person and generate customized ice breakers. It aims to provide users with interesting conversation starters tailored to the individual they are trying to connect with.

Features
Crawls Linkedin and Twitter data: The application utilizes APIs to gather relevant information about a person from their Linkedin and Twitter profiles.
Personalized ice breakers: Based on the gathered data, the application generates customized ice breakers that can be used to start conversations with the target individual.
Easy-to-use interface: The web application provides a user-friendly interface where users can input the person's details and receive the generated ice breakers.
